### PR TITLE
AMD Distribution

### PR DESCRIPTION
**Pull request by Cleanshooter**
I'm trying to reproduce an issue I'm experiencing with TinyMCE and it would be helpful if I could include TinyMCE React as an AMD module.

### CODE REVIEW

**Comment by Cleanshooter** [comment]
feel free to ignore last commit... I didn't think updating the branch would automatically update this request.

**Comment by Cleanshooter** [comment]
been trying to test the AMD module in a JS Fiddle all day and it's not working... not sure this works.  Ideas?

**Comment by tiny-james** [comment]
If you wish this PR to be considered please remove the compiled JS files.

File: package.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "ts-loader": "^5.3.3",
     "tslint": "^5.12.1",
     "typescript": "^3.2.4",
+    "uglify-js": "^3.5.11",
```
Comment (tiny-james) [inline package.json:64]: This dependency doesn't seem to be used.

**Comment by tiny-james** [comment]
Closing as compiled Js files were not removed.

**Pull request closed without merging**